src/components/PageHeader/PageHeader.stories.ts

```diff
@@ -0,0 +1,39 @@
+import { PageHeader } from './PageHeader';
+import type { Meta, StoryObj } from '@storybook/react';
+
+const meta = {
+	title: 'Components/PageHeader',
+	component: PageHeader,
+} satisfies Meta<typeof PageHeader>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+const defaultProps = {
+
```

Comment:
Please add some demo menu items here so that an example of how a real use case looks and works can be seen in Storybook.

Reply:
I’m currently encountering some issues while adding demo items for the PageHeader component in Storybook. The main issue is that the component is failing to render properly due to an error caused by missing routing context. Specifically, it seems related to React.useContext(...) returning null when trying to destructure the future property. This is likely because the component needs to be wrapped in a routing provider like BrowserRouter from React.

Given these challenges, I think it would be a good idea to move the route functionality to Redback UI as a separate ticket since NavLinks might be needed again in the future. Could we update the current ticket accordingly?

![image](https://github.com/user-attachments/assets/e68f03f7-8419-4062-b785-628019e99957)

Reply:
Thank you for raising this and the detailed explanation @Narinder788! I wonder if there's a way we can mock Router NavLinks in Storybook, or something like that - they will definitely be needed again the future, @LawrenceQiu is including them in his footer component for example. 

I will take a closer look when I have some time, but in the meantime yes I think this is a good candidate for a separate follow-up ticket. Please go ahead and create it in Planner :)

Reply:
Thank you, I resolved all the merge conflicts and will make a ticket for the issue